managing_providers/_topics/editing_a_containers_provider.md

```diff
@@ -70,6 +70,8 @@ Edit information about a provider such as the name, hostname, IP address or port
 
 7.  Under **Endpoints** in the **Alerts** tab, configure the following for Prometheus alerting from the cluster.
 
+    **Note:** Prometheus alerts are not supported on Red Hat OpenShift Container Platform version 4.
+
```

Comment:
@rwellon we no longer support OCPv3 at all, so I think we'd be better off deleting this entire section from the documentation